### DISCUSSION TITLE
Model-Based Systems Engineering

### DISCUSSION THREAD

**Discussion by 0xIonRod** [LunCoSim/lunco-sim#11] [General]
It would be amazing to implement integration with MBSE tools like Arcadia, Epsilon3, Valispace, etc.

**Comment by philiplinden** [comment]
I think we should focus efforts on Epsilon3 integration

More notes below (I had to look these tools up)

### Arcadia
- A _method_ rather than an app or API.
- There are tools built around this method like [Capella](https://www.eclipse.org/capella/arcadia.html)
- [MBSE Arcadia method ontology elements traceability](https://www.linkedin.com/pulse/mbse-arcadia-method-ontology-elements-traceability-helder-castro/)
- I found an interesting step-by-step blog article series on the topic
   - Part 1: [MBSE with Arcadia method step-by-step: Operational Analysis](https://www.linkedin.com/pulse/mbse-arcadia-method-step-by-step-helder-castro/?trackingId=P53d%2FO7w9kmURcPB5XxMOQ%3D%3D)
   - Part 2: [MBSE with Arcadia method step-by-step: System Analysis](https://www.linkedin.com/post/edit/7030815748453203968/)
   - Part 3: [MBSE with Arcadia method step-by-step: Logical Architecture](https://www.linkedin.com/pulse/mbse-arcadia-method-step-by-step-logical-architecture-helder-castro)
   - Part 4: [MBSE with Arcadia method step-by-step: Physical Architecture](https://www.linkedin.com/pulse/mbse-arcadia-method-step-by-step-physical-helder-castro)

### Valispace
- An app for requirements management, ie keeping track of system requirements and how they relate to one another.
- I don't see anything on [their website](https://www.valispace.com/) about an API or integration support.

### Epsilon3
- This is an app/software designed to streamline procedures.
- Procedures are often long, complex, detailed, and mission-critical for space systems. A single operator error can destroy an entire mission.
- "Epsilon3 websocket APIs and webhooks provide real-time updates for running procedures, viewing percent complete, etc." ([source](https://www.epsilon3.io/api-integrations)) 
-
